dashboard/app/controllers/api_controller.rb

```diff
@@ -100,6 +100,9 @@ def import_clever_classroom
     course_id = params[:courseId].to_s
     course_name = params[:courseName].to_s
 
+    if course_name.length > 255
+      course_name = course_name.truncate(255)
+    end
```

Comment:
Not a change request, just a suggestion:
```suggestion
    course_name_limit = CleverSection.column_for_attribute(:name).limit
    course_name = course_name.truncate(course_name_limit) if course_name.length > course_name_limit

```